server/util/tokenBalanceCache.ts

```diff
@@ -78,7 +75,7 @@ export default class TokenBalanceCache extends JobRunner<CacheT> {
       }));
 
     try {
-      const tokensFromListsResponses = await this._listCache.getTokens();
+      const tokensFromListsResponses = await models.Token.findAll();
```

Comment:
Is there a way we can rework the logic in this file to avoid storing all the tokens in memory, even though we save them in the db?